ArduCopter/mode_guided.cpp

```diff
@@ -911,24 +834,13 @@ void ModeGuided::pause_control_run()
     pos_control->update_z_controller();
 
     // call attitude controller
-    attitude_control->input_thrust_vector_rate_heading(pos_control->get_thrust_vector(), 0.0);
+    attitude_control->input_thrust_vector_heading(pos_control->get_thrust_vector(), auto_yaw.get_heading());
```

Comment:
@lthall I'm going to drop this change, I think it was one of yours. As it currently is it might endup just rotating on the spot if not in auto yaw hold mode. Of course we could put it into that mode when we start the pause, but then you loose your original mode when you un-pause again.

Reply:
Yep, I am happy with this. I wasn't sure if we should be allowing pilot override here. If in doubt don't change the behaviour :)